pkg/kptfile/pkgfile_test.go

```diff
@@ -96,30 +97,6 @@ upstream:
 	assert.Equal(t, KptFile{}, f)
 }
 
-// TestReadFile_failUnmarshal verifies an error is returned if the file contains any unrecognized fields.
```

Comment:
Removed this test because openAPI will have lot of unrecognized fields as it is unstructured. Let me know your thoughts on better way to handle this.

Reply:
I'm not sure I follow. What is the problem?

Reply:
I don't follow, the part of unstructured is kind of confusing.